juniper_official/chassis/check-psm-power-usage-state.rule

```diff
@@ -1,20 +1,21 @@
 /*
  * Monitors PSM power usage,state and notifies when anomalies are found
  * 
- * Two inputs control detection
+ * Three inputs control detection
  *
- *   1) "psm-power-threshold" is the threshold that causes the
+ *   1) "psm-power-high-threshold" is the threshold that causes the
  *      rule to report an anomaly. By default it's 80.
- * 
- *   2) "psm" is the power module,it can be psm0 or psm1.
+ *   2) "psm-power-low-threshold" is the threshold that causes the
+ *      rule to report an anomaly. By default it's 50. 
+ *   3) "psm" is the power module,it can be psm0 or psm1.
  *      
  */
 healthbot {
     topic hardware.chassis {
         rule check-psm-power-usage-state {
             /*
              * Monitors PSM power usage,state and notifies via dashboard, red color 
-             * when threshold is exceeded else green.
+             * when high threshold is exceeded,yellow when low is exceeded else green.
```

Comment:
Red is missing in comment

Reply:
Red color is included :
             * Monitors PSM power usage,state and notifies via dashboard, red color 
             * when high threshold is exceeded,yellow when low is exceeded else green.